packaging/debian/syslog-ng-mod-http.install

```diff
@@ -1 +1,2 @@
 usr/lib/syslog-ng/*/libhttp.so
+usr/share/syslog-ng/include/scl/telegram/*
```

Comment:
Generally users have `@include "scl.conf"` in their configuration, because we ship the default config with it. As a result they would include this scl unconditionally. Due to the requires module, users without http module would experience similar warning as below when syslog-ng starts, and they do not have any good way to eliminate it:
```
[2018-06-04T18:53:14.168695] Included file was skipped because of a missing module; module='http', location='[...]/syslog-ng/include/scl/telegram/telegram.conf:23:11'
```
Personally I like the idea to couple them together into the same deb package, because they are coupled in code level. For these reasons my preference is to leave the include there.

Reply:
Then it should be under modules and not in the scl folder, but we don't have precedence nor conventions to do that.

I consider the scl folder also add a means to discover functionality and also to use them as examples and it can't serve this purpose of we hide them.

I would probably make the quoted warning a debug message instead, maybe a verbose.

Btw, we have the java ones there as well.

Reply:
Yes, we install scl-s to a common place (and I think it's fine).
I think every module should be a self-contained module which means that it should contain the scl.
Maybe I'm wrong, but : 
 * each scl file can define its dependencies via the `@requires` keyword and I think 
 * each `requires` should be expressed by `Depends` in the Debian control file. 

So when someone installs the syslog-ng-mod-telegram package it should depend on the `syslog-ng-mod-http` package and the telegram scl file should contain the `@requires mod-http`.

Reply:
But as telegram dst is implemented as a wrapper around http dst, I think the current implementation is fine.